# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/xconsole-stropts.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./xconsole.c~	2005-09-26 21:48:20.000000000 +0200
+++ ./xconsole.c	2005-09-27 09:17:26.000000000 +0200
@@ -157,6 +157,12 @@
  */
 #  define USE_PTS 
 # endif
+# if defined (__UCLIBC__)
+/*
+ * uClibc lacks support of stropts.h
+ */
+#   undef USE_PTS
+# endif
 #endif
 
 #if defined(_AIX)
